File: flutter-frontend/lib/models/board_game.dart
class BoardGame {
  final int id;
  final String title;
  final String description;
  bool isFavorite;
  final String imageUrl;

  BoardGame({
    required this.id,
    required this.title,
    required this.description,
    this.isFavorite = false,
    required this.imageUrl,
  });

  factory BoardGame.fromJson(Map<String, dynamic> json) {
    return BoardGame(
      id: json['id'] ?? 0,
      title: json['title'] ?? '',
      description: json['description'] ?? '',
      imageUrl: json['image_url'] ?? '',
    );
  }
}

File: backend-go/internal/models/board_game.go
package models

import "time"

type BoardGame struct {
	Id          int       `json:"id"`
	Title       string    `json:"title" binding:"required"`
	Description string    `json:"description" binding:"required"`
	ImageUrl    string    `json:"image_url"`
	CreatedAt   time.Time `json:"created_at"`
}
